superset-frontend/src/SqlLab/components/ResultSet/ResultSet.test.tsx

```diff
@@ -1,271 +1,271 @@
-  query: failedQueryWithErrors,
-};
-const newProps = {
-  query: {
-    cached: false,
-    resultsKey: 'new key',
-    results: {
-      data: [{ a: 1 }],
-    },
-  },
-};
-const asyncQueryProps = {
-  ...mockedProps,
-  database: { allow_run_async: true },
-};
-const asyncRefetchDataPreviewProps = {
-  ...asyncQueryProps,
-  query: {
-    state: 'success',
-    results: undefined,
-    isDataPreview: true,
-  },
-};
-const asyncRefetchResultsTableProps = {
-  ...asyncQueryProps,
-  query: {
-    state: 'success',
-    results: undefined,
-    resultsKey: 'async results key',
-  },
-};
-fetchMock.get('glob:*/api/v1/dataset/?*', { result: [] });
+// const mockedProps = {
+//   cache: true,
+//   query: queries[0],
+//   height: 140,
+//   database: { allows_virtual_table_explore: true },
+//   user,
+//   defaultQueryLimit: 1000,
+// };
+// const stoppedQueryProps = { ...mockedProps, query: stoppedQuery };
+// const runningQueryProps = { ...mockedProps, query: runningQuery };
+// const fetchingQueryProps = {
+//   ...mockedProps,
+//   query: {
+//     dbId: 1,
+//     cached: false,
+//     ctas: false,
+//     id: 'ryhHUZCGb',
+//     progress: 100,
+//     state: 'fetching',
+//     startDttm: Date.now() - 500,
+//   },
+// };
+// const cachedQueryProps = { ...mockedProps, query: cachedQuery };
+// const failedQueryWithErrorMessageProps = {
+//   ...mockedProps,
+//   query: failedQueryWithErrorMessage,
+// };
+// const failedQueryWithErrorsProps = {
+//   ...mockedProps,
+//   query: failedQueryWithErrors,
+// };
+// const newProps = {
+//   query: {
+//     cached: false,
+//     resultsKey: 'new key',
+//     results: {
+//       data: [{ a: 1 }],
+//     },
+//   },
+// };
+// const asyncQueryProps = {
+//   ...mockedProps,
+//   database: { allow_run_async: true },
+// };
+// const asyncRefetchDataPreviewProps = {
+//   ...asyncQueryProps,
+//   query: {
+//     state: 'success',
+//     results: undefined,
+//     isDataPreview: true,
+//   },
+// };
+// const asyncRefetchResultsTableProps = {
+//   ...asyncQueryProps,
+//   query: {
+//     state: 'success',
+//     results: undefined,
+//     resultsKey: 'async results key',
+//   },
+// };
+// fetchMock.get('glob:*/api/v1/dataset/?*', { result: [] });
 
-const middlewares = [thunk];
-const mockStore = configureStore(middlewares);
-const setup = (props?: any, store?: Store) =>
-  render(<ResultSet {...props} />, {
-    useRedux: true,
-    ...(store && { store }),
-  });
+// const middlewares = [thunk];
+// const mockStore = configureStore(middlewares);
+// const setup = (props?: any, store?: Store) =>
+//   render(<ResultSet {...props} />, {
+//     useRedux: true,
+//     ...(store && { store }),
+//   });
 
-describe('ResultSet', () => {
-  test('renders a Table', async () => {
-    const { getByTestId } = setup(mockedProps, mockStore(initialState));
-    const table = getByTestId('table-container');
-    expect(table).toBeInTheDocument();
-  });
+// describe('ResultSet', () => {
```

Comment:
Do we need to comment all of this out? If you do:

```suggestion
describe.skip('ResultSet', () => {
```

It will skip all the tests.

Reply:
Thanks, yeah I'm just trying to test to see which test is OOMing. :|

Reply:
I'll revert this once I have it solved. 👍